README.md

```diff
@@ -8,17 +8,15 @@ These tutorials use the [reStructuredText](http://www.sphinx-doc.org/en/stable/r
 
 This repository is currently built automatically by two systems:
 - [![Travis Status](https://travis-ci.org/ros-planning/moveit_tutorials.svg?branch=master)](https://travis-ci.org/ros-planning/moveit_tutorials) [Github Pages + Travis](https://ros-planning.github.io/moveit_tutorials/): latest
+- ROS Melodic Build Farm: TODO, unreleased
 - [![ROS Kinetic Build Farm Status](http://build.ros.org/buildStatus/icon?job=Kdoc__moveit_tutorials__ubuntu_xenial_amd64)](http://build.ros.org/job/Kdoc__moveit_tutorials__ubuntu_xenial_amd64/) [ROS Kinetic Build Farm](http://docs.ros.org/kinetic/api/moveit_tutorials/html/)
 - [![ROS Indigo Build Status](http://build.ros.org/buildStatus/icon?job=Idoc__moveit_tutorials__ubuntu_trusty_amd64)](http://build.ros.org/job/Idoc__moveit_tutorials__ubuntu_trusty_amd64/) [ROS Indigo Build Farm](http://docs.ros.org/indigo/api/moveit_tutorials/html/)
 
-
-
-
 ## Versions
 
-- ``indigo-devel`` branch should be considered for the most part "frozen" for historical reasons
-- ``kinetic-devel`` branch is latest
-- ``melodic-devel`` branch is coming soon, see [issue](https://github.com/ros-planning/moveit_tutorials/issues/230)
+- ``indigo-devel``
+- ``kinetic-devel``
```

Comment:
Note that this is the stable version and Melodic is still in Beta

Reply:
i'd shy away from calling this beta as melodic was released last May

Reply:
technically all of moveit is in beta ;-)

i've added that kinetic "is stable" to imply that melodic is not. melodic now says "latest, changes should target this branch"